app/models/referential.rb

```diff
@@ -159,6 +159,11 @@ def lines
       metadatas_lines
     end
   end
+  
+  def lines_outside_of_scope
+    func_scope = workbench.workbench_scopes.lines_scope(associated_lines).pluck(:objectid)
```

Comment:
Je pense qu'une façon plus simple serait:

`lines.where("id NOT IN (#{associated_lines.merge(workbench.lines).select(:id)})`